smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/AuthSchemeResolverGenerator.kt

```diff
@@ -0,0 +1,291 @@
+package software.amazon.smithy.swift.codegen
```

Comment:
This is the code generator for service-specific auth scheme resolver & auth scheme resolver parameters. The generated auth scheme resolver structure complies to SRA.